main.js

```diff
@@ -28,7 +28,7 @@ function createWindow() {
 }
 
 app.on('ready', () => {
-    pythonProcess = exec('cd backend && python app.py', (err, stdout, stderr) => {
+    pythonProcess = exec('cd Backend && python app.py', (err, stdout, stderr) => {
```

Comment:
Case-sensitiveness doesn't affect windows run environment but sure for linux it should've been considered.